### PR TITLE
Update Community page to Include link to Forum

### LINKED ISSUE #297: Link to Snapcraft Forum on Community Page.
## Summary

Should include the [Snapcraft Forum](https://forum.snapcraft.io) as a place where users can get help and get involved with the community on the [community page](https://snapcraft.io/community).

### PR DESCRIPTION
**Pull request by ryanleesipes**
My first stab at copy associated with the forum, feedback welcome.

## Done

- Added the link to the [Snapcraft Forum](https://forum.snapcraft.io) with some copy.

## Issue / Card

Addresses [this issue](https://github.com/canonical-websites/snapcraft.io/issues/297) raised during Snapcraft Docs Day (March 31, 2017).

Fixes #297

### CODE REVIEW

File: community/index.html [hunk 1]
```diff
@@ -22,6 +22,13 @@ <h1 class="hero-row__heading">Community and support</h1>
             <div class="equal-height--vertical-divider">
 
                 <div class="six-col box">
+                    <h3><a href="https://forum.snapcraft.io">Talk about Snapcraft on the Forum ›</a></h3>
```
Comment (anthonydillon) [inline community/index.html]: Please encode the angle bracket using `&nbsp;&rsaquo;`

Comment (anthonydillon) [inline community/index.html]: I believe forum is lowercase.

File: community/index.html [hunk 2]
```diff
@@ -22,6 +22,13 @@ <h1 class="hero-row__heading">Community and support</h1>
...
+                    <p>Got ideas for how Snapcraft could be improved? Want to be an active part of the community and get involved? 
+                    Head over to the official <a class="external" href="https://forum.snapcraft.io">Snapcraft Forum</a>
```
Comment (anthonydillon) [inline community/index.html]: Snapcraft forum please. Lowercase forum.

**Comment by anthonydillon** [comment]
Also if you associate the issue in the original description with: `Fixes #297` it will automatically close the issue when this PR is merged.

**Comment by ryanleesipes** [comment]
@anthonydillon is that correct? Was unsure if that was what you were talking about with the angle bracket.

**Comment by anthonydillon** [comment]
That's right. That is the character we use for text links. Although you do not need the space between forum and `&nbsp;` as it is a space itself.

The reason for the none breaking space is to stop the angle bracket from wrapping on its own. But brings the last word with it.

**Comment by ryanleesipes** [comment]
@anthonydillon let me know if that is right.

**Pull request merged**